compiler/p/codegen/generateOMRInstructions.hpp

```diff
@@ -0,0 +1,394 @@
+/*******************************************************************************
```

Comment:
Perhaps this file should be named `GenerateOMRInstructions.hpp` - all other files in that folder start with a capital.

Reply:
I agree with capitalizing the name: `GenerateOMRInstructions.hpp`. However, using `OMR` as a prefix was deliberately not chosen because that typically indicates the file is part of an extensible class, which `GenerateInstructions.hpp` is not (see @0xdaryl's [comment above](https://github.com/eclipse/omr/pull/4062#issuecomment-505585232)).

Reply:
OpenJ9 *does* treat the header file as extensible: it want to include the OMR header file and add extra declarations which supports my suggestion of `OMRGenerateInstructions.hpp`.